tests/testthat/test-test-threads.R

```diff
@@ -113,7 +113,14 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
 
 
   test_that("check_run_times() returns NA for dry runs [BBR-TSTT-005]", {
-    expect_equal(check_run_times(mod1, .wait = F), NA)
```

Comment:
line 9 of this test file should say

```r
MODEL_DIR_BBI <- file.path(dirname(ABS_MODEL_DIR), "test-test-threads-models")
```

The whole point of this pattern to create a temp dir and be able to tell what created it (if it fails to get cleaned up).